mozilla-release/python/mozbuild/mozbuild/mozconfig.py

```diff
@@ -378,7 +378,7 @@ def _parse_loader_output(self, output):
                 current_type = None
                 continue
 
-            assert current_type is not None
+            assert current_type is not None, "Current line is: '%s'" % line
```

Comment:
I'd rather keep it. Otherwise in case of errors in mozconfig files, there's absolutely no information on what's actually wrong.